### ISSUE TITLE
Naming the Voyager window

### ISSUE THREAD

**Issue by Azzaare** [queryverse/DataVoyager.jl#30]
Would it be possible to have an optional arg to name our Voyager window? I happen to generate several windows from some datasets and it is quickly getting chaotic ><

**Comment by davidanthoff** [comment]
Good idea! The title is set [here](https://github.com/queryverse/DataVoyager.jl/blob/master/src/DataVoyager.jl#L23), a PR that makes that configurable would be welcome!

**Comment by Azzaare** [comment]
Sorry for the late answer! Thank you for the pointer to the relevant code, I should have some this week to take care of it ;)
